src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -173,6 +180,31 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         }
         this.updateContextOfActiveNotebookKernel(activeEditor);
     }
+    private async updateDebugContext(activeEditor?: INotebookEditor, interpreter?: PythonEnvironment) {
```

Comment:
Where is the second parameter case (interpreter) coming from? The only usage of this looks to just be passed an editor.